python/tvm/autotvm/task/nnvm_integration.py

```diff
@@ -0,0 +1,291 @@
+# pylint: disable=unused-variable,invalid-name
+"""
+Decorator and utilities for the integration with TOPI and NNVM
+"""
+import warnings
+
+from ... import _api_internal, tensor, placeholder, target as _target
+
+from ..util import get_const_tuple, get_func_name
+from .task import args_to_workload, dispatcher, create, register
+
+# Decorators for registering templates to topi
+# a table that records all registered dispatcher for all targets
+_REGISTED_DISPATHCER = {
+}
+
+def register_topi_compute(topi_compute, target_keys, template_keys):
+    """Register a tunable template for a topi compute function
```

Comment:
We need an example here in the docstring to demo the usage.

Add more details about what are target_keys and what are template keys